go/processor/geth/processor.go

```diff
@@ -230,18 +230,36 @@ func transactionToMessage(transaction tosca.Transaction, gasPrice tosca.Value, b
 		})
 	}
 
+	var authorizations []types.SetCodeAuthorization
+	if transaction.AuthorizationList != nil {
+		authorizations = make([]types.SetCodeAuthorization, len(transaction.AuthorizationList))
+		for idx, authorization := range transaction.AuthorizationList {
+			authorizations[idx] = types.SetCodeAuthorization{
+				ChainID: *uint256.NewInt(0).SetBytes(authorization.ChainID[:]),
+				Address: common.Address(authorization.Address),
+				Nonce:   authorization.Nonce,
+
+				// Ignore linting due to the deprecation of the V, R, S fields.
+				V: authorization.V,                                 //nolint:all
+				R: *uint256.NewInt(0).SetBytes(authorization.R[:]), //nolint:all
+				S: *uint256.NewInt(0).SetBytes(authorization.S[:]), //nolint:all
```

Comment:
=> see comment on not making those fields deprecated and please re-enable the linter here